### DISCUSSION TITLE
Add GDScript to Pandoc's highlighting

### DISCUSSION THREAD

**Discussion by donn-xx** [jgm/pandoc#10401] [Ideas]
Godot Game Engine has come a long way and it would be great if their GDScript language could be in the list of highlight languages. 

Not sure how hard this is to implement, but it's very python-like.

Here's the syntax:
https://docs.godotengine.org/en/latest/tutorials/scripting/gdscript/gdscript_basics.html

Thanks.

**Comment by silby** [comment]
See the [skylighting readme](https://github.com/jgm/skylighting?tab=readme-ov-file#adding-new-syntaxes); new syntaxes reach pandoc's highlighting when they reach skylighting via the KDE syntax highlighting framework

**Reply by donn-xx** [reply]
Hey, thanks. Will go ask there.

**Reply by donn-xx** [reply]
I see they have gd-script already! Maybe I just need to do some fancy updating!

**Reply by donn-xx** [reply]
Okay, so this is a bit weird. 
```gd-script``` is there (it's even 3 years old!) https://github.com/KDE/syntax-highlighting/blob/master/data/syntax/gd-script.xml

I managed to install pandoc 3.5 and it runs, but there's no gd-script (or any other spelling) in the output of ```pandoc --list-highlight-languages```

Any ideas?

**Reply by jgm** [reply]
I will update skylighting to include this.

**Reply by donn-xx** [reply]
Thankyou!
